Makefile

```diff
@@ -4,13 +4,13 @@ IMAGE_NAME ?= cloud-resource-operator
 OPERATOR_IMG = $(IMAGE_REG)/$(IMAGE_ORG)/$(IMAGE_NAME):v$(VERSION)
 MANIFEST_NAME ?= cloud-resources
 NAMESPACE=cloud-resource-operator
-PREV_VERSION=0.24.0
+PREV_VERSION=0.24.1
 VERSION=0.25.0
 COMPILE_TARGET=./tmp/_output/bin/$(IMAGE_NAME)
 UPGRADE ?= true
 CHANNEL ?= rhmi
 
-PREVIOUS_OPERATOR_VERSIONS="0.24.0,0.23.0"
+PREVIOUS_OPERATOR_VERSIONS="0.24.1,0.24.0,0.23.0"
```

Comment:
Don't recall having to do this in the passed @MStokluska Can you confirm why we need 3 versions here , is this due to the bundling ?

Reply:
that is due to the bundles and how they work. So right now if you want to be 0.25.0 you will need to update from 0.24.1 which updates from 0.24.0 which updates from 0.23.0.

Reply:
:+1:

Reply:
/lgtm
/approve